torchtext/transforms.py

```diff
@@ -322,30 +322,44 @@ class CLIPTokenizer(Module):
     (a bit like sentencepiece) so a word will be encoded differently whether it
     is at the beginning of the sentence (without space) or not.
 
```

Comment:
@abhinavarora As we discussed earlier for tokenizers doc-strings to provide information on standard out-of-the-box artifacts that we host, can we also update  to include example and provide artifacts path?

Reply:
Added in the latest commit!